src/vibratobracket.ts

```diff
@@ -72,18 +72,26 @@ export class VibratoBracket extends Element {
   draw(): void {
     const ctx = this.checkContext();
     this.setRendered();
-    const y = this.start
-      ? this.start!.getStave()!.getYForTopText(this.line)
-      : this.stop!.getStave()!.getYForTopText(this.line);
-    // If start note is not set then vibrato will be drawn
-    // from the beginning of the stave
-    const start_x = this.start ? this.start.getAbsoluteX() : this.stop!.getStave()!.getTieStartX();
+    let y = 0;
+    let start_x = 0;
+    let stop_x = 0;
+    if (this.start) {
+      y = this.start.checkStave().getYForTopText(this.line);
+      start_x = this.start.getAbsoluteX();
+    } else if (this.stop) {
+      y = this.stop.checkStave().getYForTopText(this.line);
+      // If start note is not set then vibrato will be drawn
+      // from the beginning of the stave
+      start_x = this.stop.checkStave().getTieStartX();
+    }
```

Comment:
We can simplify this:

```typescript
const y = (this.start && this.start.checkStave().getYForTopText(this.line)) ||
          (this.stop && this.stop.checkStave().getYForTopText(this.line)) || 0;

const start_x = (this.start && this.start.getAbsoluteX()) || (this.stop && this.stop.checkStave().getTieStartX()) || 0;
```

Reply:
done, I would have never expected that to return a number. Looks like a boolean expression!